### ISSUE TITLE
Overridden brandImage(for:) is not called in subclass

### ISSUE THREAD

**Issue by mahavi** [stripe/stripe-ios#1827]
## Summary
Overridden method `open class func brandImage(for cardBrand: STPCardBrand) -> UIImage?` is not called in subclass.

## Code to reproduce
```
import Stripe
import UIKit

final class PaymentCardTextField: STPPaymentCardTextField {
    override class func brandImage(for cardBrand: STPCardBrand) -> UIImage? {
        nil
    }
}
```

## iOS version
13.x

## Installation method
SPM

## SDK version
21.5.1

## Other information
I guess, the issue is with the incorrect method `brandImage(for:)` call inside `STPPaymentCardTextField` - should be `Self.brandImage(for: )` instead of `STPPaymentCardTextField.brandImage(for:)`.

**Comment by yuki-stripe** [comment]
Thanks for raising this @mahavi, we'll get it into the next release

**Comment by davidme-stripe** [comment]
We fixed this in [version 21.7.0](https://github.com/stripe/stripe-ios/releases/tag/21.7.0).
